core/src/genesis.rs

```diff
@@ -352,11 +352,20 @@ impl RawGenesisBlockBuilder {
             },
         }
     }
+
     /// Create a domain and return a domain builder which can
     /// be used to create assets and accounts.
-    pub fn domain(mut self, domain_name: Name) -> RawGenesisDomainBuilder {
+    pub fn domain(self, domain_name: Name) -> RawGenesisDomainBuilder {
+        self.domain_meta(domain_name, Default::default())
+    }
+
+    pub fn domain_meta(
+        mut self,
+        domain_name: Name,
+        metadata: iroha_data_model::prelude::Metadata,
```

Comment:
full path is redundant here because we already imported prelude on top of the module